docs/formats/zst.md

```diff
@@ -213,7 +213,7 @@ In other words, the reassembly record of the super column
 combined with the N reassembly records collectively define the original sequence
 of Zed data values in the original order.
 Taken in pieces, the reassembly records allow efficient access to sub-ranges of the
-rows, to subset of columns of the rows, to sub-ranges of columns of the rows, and so forth.
+rows, to subsets of columns of the rows, to sub-ranges of columns of the rows, and so forth.
```

Comment:
Maybe this could also be "...to a subset of columns..." but based on everything else in this paragraph, I assumed plural was the intent.